src/main/resources/view/HelpWindow.fxml

```diff
@@ -19,27 +23,48 @@
         <URL value="@Fonts.css" />
         <URL value="@HelpWindow.css" />
       </stylesheets>
-
-      <HBox alignment="CENTER">
-        <children>
-          <Label fx:id="helpMessage" text="Label">
-            <HBox.margin>
-              <Insets right="5.0" />
-            </HBox.margin>
-          </Label>
-          <Button fx:id="copyButton" mnemonicParsing="false" onAction="#copyUrl" text="Copy URL">
-            <HBox.margin>
-              <Insets left="5.0" />
-            </HBox.margin>
-          </Button>
-        </children>
-        <opaqueInsets>
-          <Insets bottom="10.0" left="5.0" right="10.0" top="5.0" />
-        </opaqueInsets>
-        <padding>
-          <Insets bottom="10.0" left="5.0" right="10.0" top="5.0" />
-        </padding>
-      </HBox>
+         <VBox minWidth="550.0">
+            <children>
+               <TableView fx:id="table" fixedCellSize="0.0" layoutY="-1.0" minHeight="500.0" minWidth="550.0" VBox.vgrow="ALWAYS">
+                 <columns>
+                   <TableColumn fx:id="action" prefWidth="111.0" reorderable="false" text="Action" />
+                   <TableColumn fx:id="format" prefWidth="232.0" reorderable="false" text="Format" />
+                     <TableColumn fx:id="example" prefWidth="229.0" reorderable="false" text="Example" />
+                 </columns>
+                  <opaqueInsets>
+                     <Insets />
+                  </opaqueInsets>
+               </TableView>
+            <HBox alignment="CENTER" layoutX="1.0" layoutY="536.0" minHeight="80.0" minWidth="550.0" style="-fx-background-color: D3DAFB50;">
+              <children>
+                <Label fx:id="helpMessage" text="Label" textFill="#424874" wrapText="true" HBox.hgrow="ALWAYS">
+                    <padding>
+                       <Insets bottom="5.0" left="10.0" right="10.0" top="5.0" />
+                    </padding>
+                </Label>
+                <Button fx:id="copyButton" mnemonicParsing="false" onAction="#copyUrl" text="Copy URL" textFill="WHITE" wrapText="true">
```

Comment:
is the copy url button suppose to change size when hover over? maybe it'll be better if it doesn't bcos it makes the text move. But then after clicking on a table cell, it doesn't change size on hovering until its clicked on...